events/examples/event-title/meetup.md

```diff
@@ -0,0 +1,35 @@
+# Introduction
+Awesome that you are interested in hosting an event!
+
+This is a template that we use here at Stockholm React JS.
+You can add / remove text from it but please try to keep the structure of it intact.
```

Comment:
I think we could be even clearer here:

"Since we parse this file programatically, please do not change headers or bold text, but feel free to change any text inside of a section or add anything you find appropriate."

Or if we decide to keep the `{{}}` in (which I'd argue against), we could say that only parts surrounded by `{{}}` should be changed.